### PR TITLE
completed css-box-model exercise - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-28 at 5 41 21 PM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 6px;
```
Comment (shawnkost) [inline css-box-model/styles.css]: You're also missing a `padding-bottom: 6px`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Bryan! Just missing `padding-bottom`

File: css-box-model/styles.css (hunk 1 above)
Comment (bbrown949) [reply css-box-model/styles.css]: fixed.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**